### PR TITLE
Notes-fixes: style, responsiveness and fixed resource

### PR DESCRIPTION
**Pull request by madvincy**
level1notephone
noteslv1

### CODE REVIEW

File: src/summary/history/encounter.resource.tsx [hunk 1]
```diff
@@ -11,3 +11,17 @@ export function getEncounters(
...
 }
+export function getEncounterById(encounterId: string) {
+  return openmrsFetch(`/ws/fhir/Encounter?${encounterId}`);
+}
+export function getEncounterByUuid(encounterUuid: string) {
+  return openmrsFetch(`/ws/fhir/Encounter?_id=${encounterUuid}`);
+}
+export function searchEncounterByPatientIdentifierWithMatchingVisit(
+  patientIdentifer: string,
+  Visit: string
```
Comment (fatmali) [inline src/summary/history/encounter.resource.tsx]: What's `Visit`? Is it `visitUuid`? Also it's convention to name variables with small caps

File: src/summary/history/notes-helper.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    return author.toUpperCase();
+  } else if (note.extension) {
+    author = note.extension.find(ext => ext.url === "changedBy");
+    return author ? author.valueString.toUpperCase() : "";
+  }
+}
+export function getNotes(notes) {
+  return notes.map(note => note.resource);
+}
+export function formatNotesDate(date: string): string {
```
Comment (fatmali) [inline src/summary/history/notes-helper.ts]: is this the same implementation as that of `formatDate` in dimension-helper?

File: src/summary/history/notes-card-style.css [hunk 3]
```diff
@@ -1,18 +1,32 @@
...
+  border: none;
+  padding: 0.625rem 1.625rem 0.625rem 1rem;
+  background-color: var(--omrs-color-bg-medium-contrast);
+  color: var(--omrs-color-ink-medium-contrast);
+  cursor: pointer;
+  text-decoration: none;
 }
 
-.tableRow {
+.tableNotesRow {
```
Comment (fatmali) [inline src/summary/history/notes-card-style.css]: I dont see why you had to rename the classes here

Comment (madvincy) [reply src/summary/history/notes-card-style.css]: It tends to be affected by other classes such as dimensions

Comment (madvincy) [reply src/summary/history/notes-card-style.css]: so making it achieve the Figma design responsiveness on phones affects other files classes. i.e display: block;

File: src/summary/history/notes-helper.ts (hunk 2 above)
Comment (madvincy) [reply src/summary/history/notes-helper.ts]: As per Figma designs date display format for notes is slightly different

File: src/summary/history/encounter.resource.tsx (hunk 1 above)
Comment (madvincy) [reply src/summary/history/encounter.resource.tsx]: noted

File: src/summary/history/notes-helper.ts (hunk 2 above)
Comment (madvincy) [reply src/summary/history/notes-helper.ts]: datesdiff
vitalsheight
this is the differents for the dimension the year starts when its less than this year

**Comment by jonathandick** [comment]
Please fix conflicts and then we can merge.

**Pull request merged**